x-pack/test/functional/apps/infra/tour.ts

```diff
@@ -36,11 +33,37 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
     after(async () => {
       await esArchiver.unload('x-pack/test/functional/es_archives/infra/metrics_and_logs');
-      await browser.removeLocalStorageItem(observTourActiveStorageKey);
       await browser.removeLocalStorageItem(observTourStepStorageKey);
     });
 
     describe('Tour enabled', () => {
+      beforeEach(async () => {
```

Comment:
I'm not sure if it makes sense to keep these tests around once https://github.com/elastic/kibana/issues/138568 is implemented, but thought it would be good to maintain them for now until we have a clearer picture of what our E2E testing looks like.